src/filesystem_common.cpp

```diff
@@ -124,7 +124,12 @@ bool blacklist_pattern_list::match_dir(const std::string& name) const
 
 std::string get_prefs_file()
 {
-	return get_user_config_dir() + "/preferences";
+	return get_sync_dir() + "/preferences";
```

Comment:
Not a request for a change, but making it a review comment will make Github create a discussion thread.

I looked at where the `user_config_dir` is still used, and it seems to only be used for the two below. It probably needs to stay for those, so can't be removed in this PR.
* remember the location of wesnothd.exe (lan_server.cfg)
* storing passwords (credentials-aes)

Reply:
I wasn't planning to move them, but also I'm not really sure how an encrypted credentials file counts as configuration in the first place. Then at that point it'd feel a bit silly to have just a single file (`lan_server.cfg`) put there.